extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -206,18 +202,8 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
     return !!draftId.match(this.localDraftPrefix);
   }
 
-  private localDraftCreate = async (mimeMsg: string, threadId: string, recipients: string[], subject: string) => {
+  private localDraftCreate = async (mimeMsg: string, threadId: string) => {
     const draftId = `${this.localDraftPrefix}${threadId}`;
-    const draftStorage = await AcctStore.get(this.view.acctEmail, ['drafts_reply', 'drafts_compose']);
-    if (threadId) { // it's a reply
-      const drafts = draftStorage.drafts_reply || {};
-      drafts[threadId] = draftId;
-      await AcctStore.set(this.view.acctEmail, { drafts_reply: drafts });
-    } else { // it's a new message
-      const drafts = draftStorage.drafts_compose || {};
-      drafts[draftId] = { recipients, subject, date: new Date().getTime() };
-      await AcctStore.set(this.view.acctEmail, { drafts_compose: drafts });
-    }
     await storageLocalSet({ [draftId]: { message: { raw: Buf.fromUtfStr(mimeMsg).toBase64UrlStr(), threadId } } });
```

Comment:
I would like to review this in another issue and another PR - I'd rather have a more structured place where to save drafts.

For example a "localDrafts" array or object where all the drafts would be.